jetty-server/src/test/java/org/eclipse/jetty/server/handler/SecuredRedirectHandlerTest.java

```diff
@@ -126,11 +126,7 @@ public static void startServer() throws Exception
         contextHandlers.setHandlers(new Handler[]{redirectHandler, rootContext, test1Context, test2Context});
 
         // Create server level handler tree
-        HandlerList handlers = new HandlerList();
-        handlers.addHandler(contextHandlers);
-        handlers.addHandler(new DefaultHandler()); // round things out
-
-        server.setHandler(handlers);
+        server.setHandler(new HandlerList(contextHandlers, new DefaultHandler()));
```

Comment:
We shouldn't get rid of _all_ of our usages of `addHandler(Handler)`.
That needs to be tested as well.

Reply:
There are still many usage examples and tests.  Specially when the handlers are complex or variable.